### DISCUSSION TITLE
Can you sort cards?

### DISCUSSION THREAD

**Discussion by jwillinghalpern** [keystonejs/keystone#7656] [Questions] [answered]
Can cards be sorted by field(s) contents in the Admin UI?

I would also be open to changing the default sort order returned by the graphql API for that list if needed.

<img width="328" alt="Screen Shot 2022-06-22 at 12 57 30 PM" src="https://user-images.githubusercontent.com/16845102/175125052-f1b380a3-4e4d-4533-ae1b-26bf9bc4898c.png">

**Comment by raveling** [comment, marked as answer]
Unfortunately at the moment this isn't possible. We’d like to support this in the future.

**Reply by jschuur** [reply]
Looks like the same applies to the `select` `displayMode` drop down entries in a relationship?

I have a list of a few hundred entries and the order looks random (sorted alphabetically by the id guid maybe?) and it looks like it's being dynamically loaded in as I scroll.
